src/components/ha-cover-controls.html

```diff
@@ -15,14 +15,28 @@
     </style>
 
     <div class='state'>
-      <paper-icon-button icon="mdi:arrow-up" on-tap='onOpenTap'
-        invisible$='[[!entityObj.supportsOpen]]'
-        disabled='[[entityObj.isFullyOpen]]'></paper-icon-button>
+      <template is='dom-if' if='[[entityObj.assumeState]]'>
+        <paper-icon-button icon="mdi:arrow-up" on-tap='onOpenTap'
+          invisible$='[[!entityObj.supportsOpen]]'></paper-icon-button>
+      </template>
+      <template is='dom-if' if='[[!entityObj.assumeState]]'>
```

Comment:
Instead of duplicating the dom element - make a function that will decide whether the button should be disabled.